refinery/data_set_manager/views.py

```diff
@@ -367,7 +369,11 @@ def post(self, request, *args, **kwargs):
                 try:
                     response = import_by_file(f)
                 except Exception as e:
-                    logger.error(traceback.format_exc(e))
+                    etype, value, tb = sys.exc_info()
+                    logger.error("{} {}".format(
+                        PARSER_UNEXPECTED_ERROR_MESSAGE,
+                        "".join(traceback.format_exception(etype, value, tb))
+                    ))
```

Comment:
Why not just `logger.error(e)`?

Reply:
I just tried to replicate the behavior - I did not try to change what it did.